language_bindings/rust/apps/iir_blur/src/main.rs

```diff
@@ -0,0 +1,83 @@
+#![allow(non_upper_case_globals)]
+#![allow(non_camel_case_types)]
+#![allow(non_snake_case)]
+
+include!(concat!(env!("OUT_DIR"), "/iir_blur.rs"));
+
+use halide_runtime::runtime_bindings::{halide_buffer_t, halide_type_t}; //todo rust enum type to runtime mod and possibly escape from useing buffer_t
+use halide_runtime::HalideBuffer;
+use image::io::Reader;
+
+fn main() {
+    println!("halide mainish thing");
+
+    let img = Reader::open("images/Hummingbird.jpg")
+        //Img source: https://commons.wikimedia.org/wiki/File:Hummingbird.jpg#filelinks
+        //Image released into public domain by Jon Sullivan PDPhoto.org
```

Comment:
I would put it in a `LICENSE` file in `language_bindings/rust` using the [Debian format](https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/):

```
Files: apps/iir_blur/images/Hummingbird.jpg
Copyright: Public Domain
Comment: https://commons.wikimedia.org/wiki/File:Hummingbird.jpg#filelinks
License: public-domain
 This work has been released into the public domain by its author, Jon Sullivan
 PD Photo.org. This applies worldwide. In some countries this may not be legally
 possible; if so: Jon Sullivan PDPhoto.org grants anyone the right to use this
 work for any purpose, without any conditions, unless such conditions are
 required by law.
```